docs/templates/PACKAGE_MAIN_DOCUMENTATION.MD

```diff
@@ -0,0 +1,105 @@
+**📦 Resources**
+
+<!-- RESOURCES:
+* required
+
+Listing of resources related to the topic including running versions of used code snippets or other applications. 
+Resource links are prefixed with the "🔗" icon
```

Comment:
are those internal or external resources?

Reply:
It was meant to link to the demo repo. I started to create a demo repo per package.